c_src/erocksdb_db.cc

```diff
@@ -1378,16 +1384,18 @@ Put(
                 !enif_inspect_binary(env, argv[3], &value))
             return enif_make_badarg(env);
         cfh = cf_ptr->m_ColumnFamily;
+        arg_opts = argv[4];
     }
     else
     {
         if(!enif_inspect_binary(env, argv[1], &key) ||
                 !enif_inspect_binary(env, argv[2], &value))
             return enif_make_badarg(env);
         cfh = db_ptr->m_Db->DefaultColumnFamily();
+        arg_opts = argv[3];
```

Comment:
need a safety check ` if (argc > 2)` and also set default if not.

Reply:
There are only `rocksdb:put/4` and `rocksdb:put/5`, so this function will not be called with less than 4 argc.